tests/deployment/switch_config.py

```diff
@@ -54,7 +61,30 @@ def server_init():
                                      'site_layout')
 
 
-class TestNativeNetwork(NetworkTest):
+@pytest.fixture
+def not_dell():
+    """open the site-layout file to see if we don't have a dell switch"""
+
+    layout_data = open('site-layout.json')
```

Comment:
In general, prefer using a `with` statement when working short-term with files:

```python
with open('filename') as f:
    data = json.load(f)
```

This way it's immediately apparent that you're not leaking the file object; you close it correctly below, but the above is more readable and less error prone.